distributed/utils.py

```diff
@@ -1608,3 +1608,14 @@ def recursive_to_dict(
         return repr(obj)
     finally:
         tok.var.reset(tok)
+
+
+def is_python_shutting_down() -> bool:
+    """Is the interpreter shutting down now?
+    This is a variant of ``sys.is_finalizing`` which can return True inside the ``__del__``
+    method of classes defined inside the distributed package.
```

Comment:
```suggestion
    method of classes defined inside the distributed package.
    """
```

Reply:
? the docstring is already closed below

Reply:
It wasn't before.  There was a bad merge from a github suggestion (my mistake).  James was correct to point it out.  I fixed it in a commit in between your two comments here. 

Anyway, all is well.